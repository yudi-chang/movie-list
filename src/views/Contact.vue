<template>
  <div>
    <h1>This is Contact page</h1>
  </div>
</template>

<style>
</style>
