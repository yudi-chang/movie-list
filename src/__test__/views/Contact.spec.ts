import { describe, it, expect } from 'vitest'
import { mount } from '@vue/test-utils'
import Contact from '@/views/Contact.vue'

describe('Contact', () => {
  const defaultParams = {}

  const createWrapper = (params = defaultParams) => {
    return mount(Contact, params)
  }

  it('should renders correctly', async () => {
    const wrapper = createWrapper();
    expect(wrapper.element).toMatchSnapshot();
  })
})
