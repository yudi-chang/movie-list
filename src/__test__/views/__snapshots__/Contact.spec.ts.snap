// Vitest Snapshot v1, https://vitest.dev/guide/snapshot.html

exports[`Contact > should renders correctly 1`] = `
<div
  class="contact-page"
  data-v-337105ab=""
>
  <div
    class="contact-container animate enter"
    data-v-337105ab=""
  >
    <div
      class="image-wrapper"
      data-v-337105ab=""
    >
      <img
        alt="Cart"
        data-v-337105ab=""
        src="/images/profile-picture.png"
      />
    </div>
    <div
      class="contact-info"
      data-v-337105ab=""
    >
      <h1
        class="text-3xl fw-bold"
        data-v-337105ab=""
      >
        M. Yudi Chang
      </h1>
      <h3
        class="text-xl fw-bold"
        data-v-337105ab=""
      >
        Frontend Developer
      </h3>
      <p
        class="text-sm"
        data-v-337105ab=""
      >
        <a
          data-v-337105ab=""
          href="mailto:m.yudi.chang@gmail.com"
        >
          m.yudi.chang@gmail.com
        </a>
      </p>
      <p
        class="text-sm"
        data-v-337105ab=""
      >
        <a
          data-v-337105ab=""
          href="https://www.linkedin.com/in/yudi-chang/"
          rel="noopener noreferrer"
          target="_blank"
        >
           https://www.linkedin.com/in/yudi-chang/ 
        </a>
      </p>
      <p
        class="text-sm"
        data-v-337105ab=""
      >
        <a
          data-v-337105ab=""
          href="tel:+6289686211228"
        >
          +6289686211228
        </a>
      </p>
    </div>
  </div>
</div>
`;
